### PR TITLE
Alex Noto

### PR DESCRIPTION
**Pull request by noto-alex-j**

### CODE REVIEW

File: test_4_filter_numbers.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 def filter_numbers(arr):
-  # Remove the `return None` code below and add your own code.
-  return None
+  newarr = [big for big in arr if big >= 10]
+  arr = [num for num in arr if num not in newarr]
```
Comment (halfghaninne) [inline test_4_filter_numbers.py:3]: Ooh! `not in` seems pretty handy! 

This solution is correct (as you know) and interesting in that it "reserves" -- albeit temporarily -- the values in `arr` that we are uninterested in. There are definitely times where we might want to preserve those values, or times that we might preserve the original collection (in your solution `arr` is redefined and not preserved). 

I'm curious how you might solve the problem in one line, without storing the "big" values we aren't concerned with.

Simply a thought experiment 😄  No indication that your work here **needs** revisiting

**Comment by halfghaninne** [comment]
Nice work 👍

File: test_4_filter_numbers.py (hunk 1 above)
Comment (noto-alex-j) [inline test_4_filter_numbers.py:3]: Haha the extra step in this code was one part me playing with list comprehension, and one part me just coding out my thought process and then not going back to revise it. I removed the newarr and replaced the code with the single line arr = [num for num in arr if num < 10]

**Comment by noto-alex-j** [comment]
Revised code!

**Pull request closed without merging**